proxy/src/coco/peer.rs

```diff
@@ -106,6 +122,69 @@ pub fn list_projects(peer: &PeerApi) -> Result<Vec<Project>, error::Error> {
         .collect()
 }
 
+/// Get all peer IDs and their branches.
+///
+/// # Errors
+///
+///   * [`error::Error::LibradLock`]
+///   * [`error::Error::Git`]
+pub fn remotes(
+    peer: &PeerApi,
+    owner: &User,
+    project_urn: &RadUrn,
+) -> Result<NonEmpty<Remote>, error::Error> {
+    let project = get_project(peer, project_urn)?;
+    let storage = peer.storage().reopen()?;
+    let repo = storage.open_repo(project.urn())?;
+
+    let (local_branches, local_tags) = with_browser(peer, &project.urn(), |browser| {
+        Ok((source::local_branches(browser)?, source::tags(browser)?))
+    })?;
+
+    let owner = owner.to_data().build()?; // TODO(finto): Dirty hack to make our Verified User into a Draft one
+    let mut remotes = NonEmpty::new(Remote {
+        user: owner,
+        branches: local_branches,
+        tags: local_tags,
+    });
+
+    for peer_id in repo.tracked()? {
+        let remote_branches = storage
```

Comment:
I'm actually going to change this to use the `Browser`

Reply:
Actually I need the 0.4.0 changes :(

Reply:
If we merge https://github.com/radicle-dev/radicle-upstream/pull/654 I can clean this up and remove the helper below